ocs_ci/deployment/cnv.py

```diff
@@ -792,3 +797,92 @@ def uninstall_cnv(self, check_cnv_installed=True):
 
         logger.info("Removing the openshift virtualization CRDs")
         self.remove_crds()
+
+    def cnv_patch_subscription(self, patch):
+        patch_cmd = (
+            f"oc -n {self.namespace} patch sub {constants.KUBEVIRT_HYPERCONVERGED} "
+            f"-p {patch} --type merge"
+        )
+        run_cmd(patch_cmd)
+
+    def get_running_cnv_version(self):
+        """
+        Get the currently deployed cnv version
+
+        Returns:
+            string: cnv version
+
+        """
+        occmd = (
+            f"oc get sub {constants.KUBEVIRT_HYPERCONVERGED} -n openshift-cnv -o json"
+        )
+        jq_cmd = "jq -r .status.currentCSV"
+        json_out = subprocess.Popen(shlex.split(occmd), stdout=subprocess.PIPE)
+        cnv_version = subprocess.Popen(
+            shlex.split(jq_cmd), stdin=json_out.stdout, stdout=subprocess.PIPE
+        )
+        json_out.stdout.close()
+        return cnv_version.communicate()[0].decode()
+
+    def check_cnv_is_upgradable(self):
+        """
+        This method checks if the cnv operator is upgradable or not
+        """
+        logger.info("Check cnv is installed")
+        assert self.cnv_hyperconverged_installed(), "cnv operator is not installed"
+
+        logger.info("Check cnv is READY")
+        assert self.post_install_verification(
+            raise_exception=False
+        ), "CNV operator is not ready"
+
+        cmd = (
+            f"oc get hyperconverged {constants.KUBEVIRT_HYPERCONVERGED} -n openshift-cnv -o json | "
+            "jq '.status.conditions'"
+        )
+        cmd_res = exec_cmd(cmd, shell=True)
+        if cmd_res.returncode != 0:
+            logger.error(f"Failed to disable multicluster engine\n{cmd_res.stderr}")
```

Comment:
This statement is not correct for this function. However, another way to improve this function is mentioned in a review comment.

Reply:
Addressed